apps/reviews/forms.py

```diff
@@ -28,9 +39,25 @@ def clean_body(self):
             raise_required()
         return body
 
+    def form_id(self):
+        return "review-reply-edit"
```

Comment:
This could be a form attribute instead: `form_id = "review-reply-edit"` at the top of the class definition